ucm/sof-sdw-rt711-1308-715/Mics.conf

```diff
@@ -0,0 +1,49 @@
+SectionDevice."Dmic" {
+	Comment "Digital Microphone"
+
+	EnableSequence [
+		cdev "hw:sofsdwrt7111308"
+		cset "name='rt715 DMIC3 Boost' 2"
+		cset "name='rt715 DMIC4 Boost' 2"
+		cset "name='rt715 ADC 24 Mux' 3"
+		cset "name='rt715 ADC 25 Mux' 4"
+		cset "name='rt715 ADC 27 Capture Switch' 1"
+		cset "name='rt715 ADC 07 Capture Switch' 1"
+		cset "name='PGA5.0 5 Master Capture Switch' 1"
+	]
+
+	DisableSequence [
+		cdev "hw:sofsdwrt7111308"
+		cset "name='PGA5.0 5 Master Capture Switch' 0"
+	]
+	Value {
+		CaptureSwitch "PGA5.0 5 Master Capture Switch"
+		CaptureVolume "PGA5.0 5 Master Capture Volume"
+		CapturePCM "hw:sofsdwrt7111308,3"
```

Comment:
@libinyang  It is defined device 4 with latest topology, and also 5,6,7 for HDMI

Reply:
First I defined the DMIC 3 and it made UCM failed. Then we got a conclusion that we should not make UCM failed, so I revert it.